rules/order.js

```diff
@@ -41,8 +67,30 @@ module.exports = {
         'justify-content',
         'align-items',
         'align-content',
-        'height',
+        'grid',
+        'grid-template-rows',
+        'grid-template-columns',
+        'grid-template-areas',
+        'grid-auto-rows',
+        'grid-auto-columns',
+        'grid-auto-flow',
+        'grid-column-gap',
+        'grid-row-gap',
+        'columns',
+        'column-gap',
+        'column-fill',
+        'column-rule',
+        'column-span',
+        'column-count',
+        'column-width',
+        'float',
```

Comment:
`float` (and I guess `clear`) is interesting, because we kind of want to discourage them. Is there any way to hint that here? Does putting them at the very bottom imply they're less useful than things at the top, for instance?

Separately, have we considered (or do we already have) a "no-floats" rule?